packages/block-editor/src/components/link-control/style.scss

```diff
@@ -59,28 +59,17 @@ $preview-image-height: 140px;
 .block-editor-link-control__field {
 	margin: $grid-unit-20; // allow margin collapse for vertical spacing.
 
-	// Element wrapping the label and input.
-	> .components-base-control__field {
-		display: flex;
-		align-items: center;
-	}
-
-	.components-base-control__label {
-		margin-right: $grid-unit-20;
-		margin-bottom: 0;
-		min-width: 29px; // align with search results.
-	}
```

Comment:
Flagging the move to stack the labels and inputs will increase the overall height of this control. This can be a problem on smaller screens.

In the past we spent a while intentionally streamlining the visual "height" to try and mitigate this.

Are we _sure_ we want to go back on this?

Reply:
As long as we're aware of the reasons as to why it was made horizontal then 👍